CondCore/Utilities/test/test_uploadConditions.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
you can use `#!/bin/bash -e` and then you not need `check_for_success and check_for_failure`. with `-e` script will fail and exit on first failed command.

Reply:
I'd rather leave it as is in order to have the printouts.

Reply:
you can use `#!/bin/bash -ex` to have the details printsouts :-)

Reply:
that doesn't work either, because this: 
https://github.com/cms-sw/cmssw/blob/499f39e604b4e3e12786228de822968be8ad1d63/CondCore/Utilities/test/test_uploadConditions.sh#L43-L54

is actually expected to fail (due to reasons explained here: https://github.com/cms-sw/cmssw/pull/34379#issuecomment-875617244). So doing what you suggest the test would constantly fail.